### PR TITLE
WIP: Honor maddr in izz ##bin

### PR DESCRIPTION
**Pull request by radare**
**Checklist**

- [ ] Closing issues: #issue
- [ ] Mark this if you consider it ready to merge
- [ ] I've added tests (optional)
- [ ] I wrote some lines in the [radare2book](https://github.com/radareorg/radare2book)

**Description**

### CODE REVIEW

**Comment by trufae** [comment]
Ill add tests. It seems to fix thw specific issue but there are still corner cases to pet

File: test/db/cmd/cmd_iz [hunk 1]
```diff
@@ -558,3 +558,32 @@ EXPECT=<<EOF
...
+0x0 24 24 hello world\nSecond Line\n
+- offset -   0 1  2 3  4 5  6 7  8 9  A B  C D  E F  0123456789ABCDEF
+0x00000000  6865 6c6c 6f20 776f 726c 640a 5365 636f  hello world.Seco
+0x00000010  6e64 204c 696e 650a ffff ffff ffff ffff  nd Line.........
+EOF
+RUN
+
+NAME=mapaddr strings
+ARGS=-m 0x1000
+FILE=bins/other/hello.txt
```
Comment (Lazula) [inline test/db/cmd/cmd_iz:578]: duplicate test names?

**Comment by radare** [comment]
Should be an error imho
